pysindy/optimizers/elastic_net.py

```diff
@@ -91,3 +93,8 @@ def _reduce(self, x, y):
 
         self.coef_ = elastic_net_model.coef_
         self.iters = elastic_net_model.n_iter_
+
+    @property
+    def complexity(self):
+        check_is_fitted(self)
+        return count_nonzero(self.coef_)
```

Comment:
What about the intercept (if `fit_intercept=True`)?

Reply:
I've added an extra term for the intercept, but in doing so I uncovered another problem. I'll create an issue and elaborate there.